sike/sike_test.go

```diff
@@ -0,0 +1,41 @@
+package sike
+
+import (
+	"bytes"
+	"crypto/rand"
+	"testing"
+)
+
+func TestP751(t *testing.T) {
+	testSIKE(NewP751SIKE(), t)
+}
+
+func BenchmarkP751SIKE(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		testSIKE(NewP751SIKE(), b)
+	}
+}
+
+func testSIKE(sikep751 SIKE, t testing.TB) {
```

Comment:
I would really like to see at least one KAT here (decapsulation). I think NIST submission contains some KATs (key, shared secret and ct).